src/main/java/com/verapdf/restapi/dto/PathDTO.java

```diff
@@ -0,0 +1,31 @@
+package com.verapdf.restapi.dto;
+
+public class PathDTO {
```

Comment:
are we still using this class? if not, then delete it

Reply:
Still using